libs/repositories/src/utils/cache.ts

```diff
@@ -1,5 +1,8 @@
 export type PartialCacheKey = string | number | boolean;
 
 export function getCacheKey(...params: PartialCacheKey[]) {
-  return params.map((param) => param.toString().toLowerCase()).join(':');
+  return params
+    .filter((item) => item !== '')
```

Comment:
handles the fact that item can be empty string

I noticed this because our key generation was adding a double `:` in the path of the cached request. This came from doing a split of a string led by `/` (like `/my/path`)